tests/test_spherePoint.py

```diff
@@ -155,8 +154,7 @@ def testInitNArgFail(self):
     def testGetLongitudeValue(self):
         """Test if getLongitude() returns the expected value.
         """
-        for lon, lat in self._dataset:
-            point = SpherePoint(lon, lat)
+        def checkGetLongitudeValue(point, lon, lat):
```

Comment:
I think this is a bit too long to treat as a lambda. Consider making a private member of `SpherePointTestSuite` for readability.

Reply:
Technically that this is not a "lambda", since Python uses the keyword `lambda` for those. But I understand and will make them methods.

Reply:
I reverted all the new in-line functions and am using better iteration instead. This really cleans up the code.